ulog/README.md

```diff
@@ -1,16 +1,15 @@
 # Logging package
 
 `package ulog` provides an API wrapper around the logging library
```

Comment:
Yeah, it's fine to provide that but we no longer wrap zap in any way though - that's old way